statement_test.go

```diff
@@ -40,6 +40,72 @@ func openConn(t *testing.T) *sql.Conn {
 	return conn
 }
 
+func TestExecStmt(t *testing.T) {
+	dqlQuery := "SELECT 1"
```

Comment:
Can we have a dml and multistatement tested here? Does it make sense?

Reply:
Done